_posts/2015-12-10-readme-refresh-day.md

```diff
@@ -55,7 +55,7 @@ Collaborators on software projects are frequently encouraged to write good insta
 ## Organizing the event
 
 A small group of existing CFPB developers organized the event, including planning and event-day activities,
-documented at [https://github.com/cfpb/readme-refresh-day](https://github.com/cfpb/readme-refresh-day).
+documented at [https://github.com/cfpb/readme-refresh-day](https://github.com/cfpb/readme_refresh_day).
```

Comment:
I was going to but didn't want to deny you the opportunity of calling me out on it. 

One must follow protocol, after all.